<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AtCoreInstanceWidget</class>
 <widget class="QWidget" name="AtCoreInstanceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>362</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="mainToolBarLayout"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="toolBarLayout"/>
   </item>
   <item>
    <widget class="QTabWidget" name="mainTab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabWidgetPage1">
      <attribute name="title">
       <string>Controllers</string>
      </attribute>
      <layout class="QVBoxLayout" name="controlTabLayout">
       <item>
       <layout class="QHBoxLayout" name="controlsLayout">
        <item>
        <widget class="BedExtruderWidget" name="bedExtWidget" native="true"/>
       </item>
      </layout>
    </item>
    </layout>
     </widget>
     <widget class="QWidget" name="advancedTab">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="advancedTabLayout">
       <item>
        <widget class="RatesControlWidget" name="ratesControlWidget" native="true"/>
       </item>
       <item>
        <widget class="PushGCodeWidget" name="pushGCodeWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="statusLayout">
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BedExtruderWidget</class>
   <extends>QWidget</extends>
   <header>bedextruderwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PushGCodeWidget</class>
   <extends>QWidget</extends>
   <header>pushgcodewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RatesControlWidget</class>
   <extends>QWidget</extends>
   <header>ratescontrolwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
